docs/using.rst

```diff
@@ -42,9 +42,9 @@ You can get the version string for ``wheel`` by running the following::
     >>> version('wheel')
     '0.32.3'
 
-You can also get the set of entry points keyed by group, such as
+You can also get a collection of entry points selectable by group, such as
```

Comment:
it's actually selectable by any of the EntryPoint attributes

```suggestion
You can also get a collection of entry points selectable by name, group or value, such as
```

Reply:
I'd probably not want to emphasize by value, as I don't expect legitimate use-cases to care about select by value. By name on the other hand seems a likely usage. Maybe.

```suggestion
You can also get a collection of entry points selectable by group or name (or other attribute), such as
```

Reply:
Or other attribute is a bit odd when there's only one other attribute

Reply:
Other possible attributes include `dist`, `module`, `attr`, and `extras`. See dbe114cbdc49.
```suggestion
You can also get a collection of entry points selectable by properties of the EntryPoint (typically 'group' or 'name'), such as
```